### DISCUSSION TITLE
DNS Resolution sporadic in SABNZBD when going through GlueTun

### DISCUSSION THREAD

**Discussion by Bugmaniac71** [qdm12/gluetun#1447] [Q&A] [answered]
Having issue with SABNZB Container. When I link in GlueTun the only way I can get it to resolve the news servers is by specifying - BLOCK_MALICIOUS=off, has anyone else experienced this please?

Regards.
Carl.

**Comment by bnhf** [comment, marked as answer]
@Bugmaniac71 

Yes. I think it's a bit better to use the environment variable UNBLOCK for the specific sites you want to be able to access (e.g. UNBLOCK=usenetserver.com) though.

**Reply by Bugmaniac71** [reply]
Ah thank you, thats fixed it!

Regards.
C
